x11/wine-crossover/Portfile

```diff
@@ -118,26 +108,49 @@ configure.args              --without-alsa \
                             --without-ldap \
                             --without-mpg123 \
                             --without-openal \
-                            --with-opengl \
                             --without-oss \
                             --with-png \
                             --with-pthread \
                             --without-sane \
                             --with-tiff \
                             --without-v4l \
-                            --with-xcomposite \
-                            --with-xcursor \
-                            --with-xinerama \
-                            --with-xinput \
+                            --without-x \
                             --with-xml \
-                            --with-xrandr \
-                            --with-xrender \
-                            --with-xshape \
-                            --with-xslt \
-                            --with-xxf86vm \
-                            --with-x \
-                            --x-include=${prefix}/include \
-                            --x-lib=${prefix}/lib
+                            --with-xslt
+
+# Wine has a native macOS driver since 1.5.26 so X11 is optional
+variant x11 {
+  depends_lib-append          port:libGLU \
+                              port:mesa \
+                              port:xorg-libX11 \
+                              port:xorg-libXcomposite \
+                              port:xorg-libXcursor \
+                              port:xorg-libXext \
+                              port:xorg-libXi \
+                              port:xorg-libXinerama \
+                              port:xorg-libXrandr \
+                              port:xorg-libXxf86vm \
+                              port:xorg-libsm \
+                              port:xorg-xf86vidmodeproto \
+                              port:xrender
+
+  configure.args-delete       --without-x
+
+  configure.args-append       --with-opengl \
+                              --with-xcomposite \
+                              --with-xcursor \
+                              --with-xinerama \
+                              --with-xinput \
+                              --with-xrandr \
+                              --with-xrender \
+                              --with-xshape \
+                              --with-xxf86vm \
+                              --with-x \
+                              --x-include=${prefix}/include \
+                              --x-lib=${prefix}/lib
+}
+
+default_variants            +x11
```

Comment:
Ah, OK, if `+x11` builds both GUIs, then this is even better. I thought that building with X11 excluded the native GUI and vice versa. In that case keeping X11 makes sense.